desugreet/member_comparator.py

```diff
@@ -0,0 +1,16 @@
+class SameMemberComparator:
```

Comment:
couldn't this class just be 2 functions? we always temporarily create it and it doesn not manage state really, it's just `input -> output`

Reply:
will be changed as discussed